doc/content/gateways/concepts/mtls-authentication/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "mTLS Authentication"
+description: ""
+---
+
+{{% tts %}} supports mutual TLS to authenticate {{% lbs %}} CUPS connections in selected deployments.
+
+<!--more-->
+
+{{% tts %}} does not natively terminate mTLS connections. This is done by a proxy. The proxy is expected to forward the sanitized client certificate chain as an HTTP header. To check it your deployment supports it, contact the operator.
+
+### Procedure
```

Comment:
```suggestion
## Procedure
```

Reply:
this is because we want this title to show on the right hand menu, like this:
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/65215579/c886e08d-42e1-4683-8ee8-cc13b3338f25)